pkg/api/controller_test.go

```diff
@@ -320,6 +320,25 @@ func TestController_CreateRepositoryHandler(t *testing.T) {
 			t.Fatalf("expected error creating duplicate repo")
 		}
 	})
+
+	t.Run("create repo with wrong storage type", func(t *testing.T) {
```

Comment:
nit; you can use the word "conflicting" instead of "wrong"?  wrong is less descriptive